### PR TITLE
Add build script for generating clang-format binaries.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REV=340070
+
+if [ ! -d llvm ]; then
+  svn co -q https://llvm.org/svn/llvm-project/llvm/trunk llvm -r $REV
+else
+  cd llvm
+  svn update -r$REV
```
Comment (yarneo) [inline build.sh]: nit: space between -r and $REV

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  cd ../
+fi
+
+cd llvm/tools
+if [ ! -d clang ]; then
+  svn co -q https://llvm.org/svn/llvm-project/cfe/trunk clang -r $REV
+else
+  cd clang
+  svn update -r$REV
```
Comment (yarneo) [inline build.sh]: nit: same as above

**Review by yarneo — APPROVED** [review]

File: build.sh (hunk 2 above)
Comment (jverkoey) [reply build.sh]: Done.

File: build.sh (hunk 1 above)
Comment (jverkoey) [reply build.sh]: Done.

**Pull request merged**